### PR TITLE
CB-233 and CB-228

### PR DESCRIPTION
**Pull request by dsafar**
Covers CB-233 and CB-228, adding three methods to WebDriverElement: executeJavascript(), .getWidth(), and .getHeight(), plus tests.

Also revs the version number and reverts the removal of the config submodule as the files contained therein are required for the test jobs.

### CODE REVIEW

File: src/main/java/com/sugarcrm/candybean/automation/element/Element.java [hunk 1]
```diff
@@ -53,9 +53,39 @@ public Element(Hook hook, int index) {
 	 */
 	public abstract Element getElement(Hook hook, int index) throws CandybeanException;
 
+	/**
+	 * Executes a javascript command against this element.
+	 * @param	javascript	The javascript code to execute
+	 * @param	javascript	The arguments to pass.  Note that indices of the arguments passed will
```
Comment (mazen-louis) [inline src/main/java/com/sugarcrm/candybean/automation/element/Element.java]: the param key is not javascript in this line, I'm guessing it was supposed to be "args"

File: src/main/java/com/sugarcrm/candybean/automation/webdriver/WebDriverElement.java [hunk 2]
```diff
@@ -217,6 +217,50 @@ public Element getElement(Hook hook, int index) throws CandybeanException {
 	}
 
 	/**
+	 * Executes a javascript command against this element.
+	 *
+	 * @param	javascript	The javascript code to execute
+	 * @param	javascript	The arguments to pass.  Note that indices of the arguments passed will
```
Comment (mazen-louis) [inline src/main/java/com/sugarcrm/candybean/automation/webdriver/WebDriverElement.java]: "args" or "javascript" ?

File: src/test/java/com/sugarcrm/candybean/automation/webdriver/WebDriverElementSystemTest.java [hunk 3]
```diff
@@ -307,4 +307,42 @@ public void sendStringTest() throws Exception {
...
+
+	@Test
+	public void getCssValueTest() throws CandybeanException {
+		iface.go("https://www.google.com/");
+		WebDriverElement searchBox = iface.getWebDriverElement(new Hook(Strategy.ID, "gbqfq"));
+		Assert.assertEquals(searchBox.getCssValue("display"), "block");
+	}
+
+	@Test
+	public void getSourceTest() throws CandybeanException {
```
Comment (mazen-louis) [inline src/test/java/com/sugarcrm/candybean/automation/webdriver/WebDriverElementSystemTest.java]: Ummm, isn't this a duplicate of the getCssValueTest() method above?

**Comment by mazen-louis** [comment]
@dsafar ready for some updates. Mostly JavaDoc issues.

**Comment by dsafar** [comment]
Fixed, ready for round 2.

**Pull request merged**